pkg/nfd-master/nfd-master.go

```diff
@@ -792,16 +838,14 @@ func (m *nfdMaster) nfdAPIUpdateOneNode(cli k8sclient.Interface, node *corev1.No
 		return objs[i].Namespace < objs[j].Namespace
 	})
 
-	klog.V(1).InfoS("processing of node initiated by NodeFeature API", "nodeName", node.Name)
-
-	features := nfdv1alpha1.NewNodeFeatureSpec()
+	klog.V(1).InfoS("processing of node initiated by", "API", api, "nodeName", nodeName)
```

Comment:
I think this is confusing log message. 

```suggestion
```

Also drop the `api` arg from the method